### PR TITLE
CMR-6289 Stabilize CSW and Opensearch Builds

### PR DESCRIPTION
**Pull request by daniel-zamora**
Upgrades rails to 5 in order to address bundle-audit failures.

### CODE REVIEW

File: app/controllers/granules_controller.rb [hunk 1]
```diff
@@ -121,7 +121,7 @@ def extract_params
   # step 2: search for granules ONLY in the collection of interest indentified in step 1
   def two_step_search
     ret_val = true
-    if params.size == 2 && params['action'] == 'index' && params['controller'] == 'granules'
+		if params.values.size == 2 && params['action'] == 'index' && params['controller'] == 'granules'
```
Comment (jaybarra) [inline app/controllers/granules_controller.rb]: extra spaces

File: config/application.rb [hunk 2]
```diff
@@ -64,7 +64,7 @@ class Application < Rails::Application
     config.assets.compress = true
     config.assets.precompile = ['*.js', '*.css', '*.css.erb', '*.png', '*.jpg', '*.jpeg', '*.gif']
     config.assets.prefix = '/assets'
-    config.relative_url_root = '/opensearch'
+    # config.relative_url_root = '/opensearch'
```
Comment (jaybarra) [inline config/application.rb]: can this be deleted instead of commented out?

File: spec/views/granules/required_form_params_spec.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
   it 'does not allow granule searches with empty UniqueID and ShortName and CollectionID' do
     VCR.use_cassette 'views/granule/allRequiredFormParamsEmpty', :record => :once, :decode_compressed_response => true do
+			puts granules_path
```
Comment (jaybarra) [inline spec/views/granules/required_form_params_spec.rb]: should this line up with the line below?

Comment (daniel-zamora) [reply spec/views/granules/required_form_params_spec.rb]: nope

File: config/application.rb (hunk 2 above)
Comment (daniel-zamora) [reply config/application.rb]: yup

File: app/controllers/granules_controller.rb (hunk 1 above)
Comment (daniel-zamora) [reply app/controllers/granules_controller.rb]: vim

File: spec/views/granules/required_form_params_spec.rb (hunk 3 above)
Comment (daniel-zamora) [reply spec/views/granules/required_form_params_spec.rb]: It should be gone, debug code.

**Pull request merged**